index.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+
+const app = express()
+
+// let roleModel = {
+//     name: "Daniele",
+//     occupation: "Engineer",
+//     languages:["Italian", "English"]
+//   }
+
+
+app.listen(5006)
+
+// app.get('/role-model', (req, res)=>{
+//     res.json(roleModel)
+// })
+
+app.get('/role-model', (req, res)=>{
+    res.json({name: "Daniele", occupation: "Engineer", languages:["Italian", "English"]})
```

Comment:
Nice approach! also you can do it by passing the roleModel 
```suggestion
 res.json(roleModel);
```